CONTRIBUTING.md

```diff
@@ -59,8 +59,10 @@ For more information on `@all-contributors` see it's [usage docs](https://allcon
 
 ### Release process:
 
+When merging a pull request to `master`, the squashed commit message should follow the [Conventional Commits](https://www.conventionalcommits.org) specification. This enables us to automatically generate CHANGELOGs & determine a semantic version bump.
```

Comment:
So the version bump happens automatically based on commit messages? Is that a lerna feature?

Reply:
Yeah, see https://github.com/lerna/lerna/tree/master/commands/version#--conventional-commits

The same can be achieved without Lerna, using [semantic-release](https://github.com/semantic-release/semantic-release)

Reply:
cool! can you just start using the conventional commits at the sametime as semantic release? or will it mess things up if you didn't do it from the start?

Reply:
I want to set this up on react-browser-hooks too

Reply:
With Lerna you can definitely start half way through and it will start to pick things up from that point. I haven't used `semantic-release` myself, but it does provide a [dry run](https://semantic-release.gitbook.io/semantic-release/usage/configuration#dryrun) option where it prints out what it would do, so it's worth giving it a go locally.